### PR TITLE
Fixed bug in CMakeLists.txt.

### PR DESCRIPTION
**Pull request by liangfok**
This bug was resulting in the following CMake warning:

```
CMake Warning (dev) in examples/SimpleCar/test/CMakeLists.txt:
  A logical block opening on the line

    /home/liang/dev/drake/drake/examples/SimpleCar/test/CMakeLists.txt:1 (if)

  closes on the line

    /home/liang/dev/drake/drake/examples/SimpleCar/test/CMakeLists.txt:5 (endif)

  with mis-matching arguments.
This warning is for project developers.  Use -Wno-dev to suppress it.

```

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2092)

### CODE REVIEW

File: drake/examples/SimpleCar/test/CMakeLists.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ if (GTEST_FOUND AND LCM_FOUND AND NOT WIN32)
   add_executable(simple_car_test simple_car_test.cc)
   target_link_libraries(simple_car_test ${GTEST_BOTH_LIBRARIES} drakeSimpleCar)
   add_test(NAME simple_car_test COMMAND simple_car_test)
-endif (GTEST_FOUND AND NOT WIN32)
+endif (GTEST_FOUND AND LCM_FOUND AND NOT WIN32)
```
Comment (sherm1) [inline drake/examples/SimpleCar/test/CMakeLists.txt]: This is fine, but you could also go crazy and change this to just
```
endif()
```
and avoid duplication of the condition altogether!

Comment (liangfok) [inline drake/examples/SimpleCar/test/CMakeLists.txt]: Good point! I've updated my PR.

**Comment by sherm1** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2092#-:-KFKdASRgm_cHGes6NYy:1417088721)*

**Comment by david-german-tri** [comment]
Opened #2100 and #2101 for the MATLAB flakes here, which are obviously unrelated.  Merging.

**Pull request merged**